### ISSUE TITLE
1.0.2 version published

### ISSUE THREAD

**Issue by Techn1x** [discourse/ember-route-template#6]
Seems to be missing from changelog / releases etc.

Not sure what was in the bump exactly. Was it a mispublish?

![Screenshot 2023-11-13 at 12 14 46 pm](https://github.com/discourse/ember-route-template/assets/1049837/bb5f7c34-687f-456f-8c07-d4f38ff08e7c)

This seems to show a diff, but those changes do not seem to be reflected in the repo here
https://app.renovatebot.com/package-diff?name=ember-route-template&from=1.0.1&to=1.0.2

```diff
-             const { addonV1Shim } = require('@embroider/addon-shim');
-
-             module.exports = addonV1Shim(__dirname);
+             // const { addonV1Shim } = require('@embroider/addon-shim');
+             // module.exports = addonV1Shim(__dirname);
+             module.exports = {
+               name: __dirname,
+             };
```

**Comment by chancancode** [comment]
huh. thanks for catching it. I published it but forgot to push. There were no intentional changes, just fixing the missing description in package.json. Not sure how the other change got in there and it was unintentional. I’ll push another release to fix it tonight

**Comment by chancancode** [comment]
Should be fixed in 1.0.3!
